b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/manageUserSection/otherComponents/addUserComponent.js

```diff
@@ -276,3 +328,16 @@ export default function AddUserComponent(prop) {
 
     );
 }
+
+function EmailInvitePanel() {
+    return (
+        <Panel bordered>
+            <Stack spacing={ 30 }>
+                <EmailFillIcon style={ { fontSize: "3em" } } />
+                An email with a confirmation link will be sent to the provided
+                email address for the user to set their own password.
+            </Stack>
+
```

Comment:
Is this new line needed? Please check other places also.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/ddf8187a9e62d909ef355a3b38a14e475a35c7aa